deployer/src/persistence.rs

```diff
@@ -135,6 +155,23 @@ impl Persistence {
     async fn get_deployment_logs(&self, name: &str) -> Result<Vec<Log>> {
         get_deployment_logs(&self.pool, name).await
     }
+
+    pub fn get_build_log_subscriber(&self) -> Receiver<BuildLog> {
+        self.build_log_send.subscribe()
+    }
+
+    pub async fn get_build_logs(&self, name: &str) -> Result<Vec<BuildLog>> {
+        let logs: Vec<Log> =
+            sqlx::query_as("SELECT * FROM logs WHERE name = ? OR state = ? ORDER BY timestamp")
+                .bind(name)
+                .bind(State::Building)
+                .fetch_all(&self.pool)
```

Comment:
Do we want an upper bound on this to prevent some crazy edge case?

Reply:
I'm not sure yet :thinking: For now I'm adding a todo comment and linear ticket to test this when all the pieces are together

Reply:
... that `OR` should be an `AND` though :man_facepalming: